simulation_parameters/GameWiki.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+///   All wiki pages to be recreated in the Thriveopedia, grouped by type.
+/// </summary>
+public class GameWiki : IRegistryType
+{
+    /// <summary>
+    ///   Unused
+    /// </summary>
+    public string InternalName { get; set; } = null!;
+
+    public Page OrganellesRoot { get; set; } = null!;
+
+    public List<Page> Organelles { get; set; } = null!;
+
+    public void ApplyTranslations()
+    {
+    }
+
+    public void Check(string name)
+    {
```

Comment:
This check method could at least verify there's at least one page, and that there is no SectionBody that is missing in any of the pages.

Reply:
Added checks for all pages.